cyberdrop_dl/scraper/crawlers/nekohouse_crawler.py

```diff
@@ -0,0 +1,270 @@
+        service, user = await self.get_service_and_user(scrape_item)
+        user_str = await self.get_user_str_from_profile(soup)
+        service_call = self.primary_base_domain / service / "user" / user
+        while offset <= maximum_offset:
+            async with self.request_limiter:
+                soup: BeautifulSoup = await self.client.get_BS4(self.domain, service_call.with_query({"o": offset}))
+                offset += 50
+                
+                posts = soup.select(self.post_selector)
+                if not posts:
+                    break
+                for post in posts:
+                    # Create a new scrape item for each post
+                    post_url = post.get("href", "")
+                    if post_url[0] == "/":
+                        post_url = post_url[1:]
+                    post_id = post_url.split("/")[-1]
+                    if post_url == "":
+                        continue
+                    post_link = self.primary_base_domain / post_url
+                    # Call on self.post to scrape the post by creating a new scrape item
+                    new_scrape_item = await self.create_scrape_item(scrape_item, post_link, "", add_parent=self.primary_base_domain / service / "user" / user)
+                    await self.post(new_scrape_item, post_id, user, service, user_str)
+
+    @error_handling_wrapper
+    async def post(self, scrape_item: ScrapeItem, post_id: int = None, user: str = None, service: str = None, user_str: str = None, unlinked_post: bool = False) -> None:
+        """Scrapes a post"""
+        if any(x is None for x in (post_id, user, service, user_str)):
+            service, user, post_id = await self.get_service_user_and_post(scrape_item)
+            user_str = await self.get_user_str_from_post(scrape_item)
+        await self.get_post_content(scrape_item, post_id, user, service, user_str, unlinked_post)
+
+    @error_handling_wrapper
+    async def get_post_content(self, scrape_item: ScrapeItem, post: int, user: str, service: str, user_str: str, unlinked_post: bool = False) -> None:
+        """Gets the content of a post and handles collected links"""
+        if post == 0:
+            return
+
+        post_url = scrape_item.url
+        if unlinked_post is True:
+            async with self.request_limiter:
+                soup: BeautifulSoup = await self.client.get_BS4(self.domain, post_url)
+                data = {
+                    "id": post,
+                    "user": user,
+                    "service": service,
+                    "title": soup.select_one(self.post_title_selector).text.strip(),
+                    "content": soup.select_one(self.post_content_selector).text.strip(),
+                    "user_str": user_str,
+                    "published": soup.select_one(self.post_timestamp_selector).text.strip(),
+                    "file": [],
+                    "attachments": []
+                }
```

Comment:
still raising `AtributeError` if the soup does not have all of these tags

Ex: This post have no content https://nekohouse.su/fantia/user/29013/post/3063417
Ex: This post have no date https://nekohouse.su/fantia_products/user/471737/post/349121